c2corg_ui/views/document.py

```diff
@@ -431,6 +431,24 @@ def get_slug_for_title(title):
     return slug
 
 
+def is_collaborative(doc):
```

Comment:
Since this function is only used in the templates, please moved it to https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/utils/__init__.py